src/main/kotlin/interpreter/pyobject/PyObject.kt

```diff
@@ -86,12 +59,9 @@ abstract class PyObject() {
             }
 
         /**
-         * Gets the default object dict, containing the base implements of certain magic methods.
+         * The default object dict, containing the base implements of certain magic methods.
          */
-        fun getDefaultDict(): LinkedHashMap<String, PyObject> {
-            val d = linkedMapOf<String, PyObject>()
-            return d
-        }
+        val defaultDict: LinkedHashMap<String, PyObject> = linkedMapOf()
```

Comment:
This is specifically not cached because it should be re-created each time. The default "empty" impl is because of a WIP.